xml/admin-ironic.xml

```diff
@@ -3575,9 +3575,6 @@ URL</literal>.</para>
                   <screen>ironic port-create -n $NODE -a &lt;MAC_address_of_Ucs_server's_NIC&gt;</screen>
                 </section>
               </section>
-              <section>
-                <title>References</title>
-              </section>
```

Comment:
This is missing a link: https://docs.openstack.org/ironic/pike/admin/drivers/ucs.html#references

Reply:
Although relaly this one doesn't matter